packages/cadl-ranch-specs/http/type/union/main.tsp

```diff
@@ -79,3 +91,75 @@ This test is testing sending the second union value in named union property.
 @route("/model2")
 @post
 op sendSecondNamedUnionValue(@body input: ModelWithNamedUnionProperty): OkResponse;
+
+@scenario
+@scenarioDoc("""
+This test is tesing receiving a string value in simple union property.
+
+Expect request:
+```json
+{ "kind": "string" }
+```
+
+Expect response:
+```json
+{ "simpleUnion": "string" }
+```
+""")
+@route("/receive/string")
+@post
+op receiveString(@body input: Input): ModelWithSimpleUnionPropertyInResponse;
+
+@scenario
+@scenarioDoc("""
+This test is tesing receiving an int array value in simple union property.
+
+Expect request:
+```json
+{ "kind": "int-array" }
+```
```

Comment:
string or int-array is already in route, why need to have it in request body?

Reply:
OK I could remove. And how to make two scenarios which one of them send the first type of union, the other send the other type of the union? Should I add another `@scenario` to this operation?